package.json

```diff
@@ -203,7 +203,7 @@
     "codegen": "apollo codegen:generate --target=typescript gqlTypes",
     "codegen-sdk": "apollo codegen:generate --target=typescript gqlTypes --config src/@sdk/apollo.config.js --globalTypesFile src/@sdk/gqlTypes/globalTypes.ts",
     "codegen-watch": "npm run codegen -- --watch",
-    "lint": "eslint src",
+    "lint": "npx eslint \"src/**/*.@(tsx|ts|jsx|js)\" --fix ; npx prettier --check \"src/**/*.@(tsx|ts|jsx|js)\" --write",
```

Comment:
It should work without `npx`

Reply:
npx is by default bundled with npm, whats wrong with it? I had some issues when trying to run it without npx (had to use node modules path)